kafka/src/test/java/no/nav/common/kafka/consumer/feilhandtering/KafkaConsumerRecordProcessorIntegrationTest.java

```diff
@@ -229,7 +229,7 @@ public void should_consume_records_if_backoff_expired() {
                         .withLockProvider(lockProvider)
                         .withKafkaConsumerRepository(consumerRepository)
                         .withConsumerConfigs(configs)
-                        .withBackoffStrategy((r) -> Duration.ZERO)
+                        .withBackoffStrategy((r) -> Duration.ofSeconds(-1))
```

Comment:
Java 21 er tydeligvis akkurat litt for kjapp til at den klarte å se denne som passert 😅